File: square.py
class Square():
    def __init__(self, location):
        self._location = location
        self._value: int = 0

    @property
    def value(self):
        return self._value

    @value.setter
    def value(self, v):
        self._value = v

    @property
    def location(self):
        return self._location

File: KenKen Solver/KenKen Solver/Square.swift
//
//  Square.swift
//  KenKen Solver
//
//  Created by Emil Bach Mikkelsen on 4/14/22.
//

struct Square: CustomStringConvertible {
    var value: Int;

    init() {
        self.value = 0;
    }
    
    var description: String {
        return "[\(self.value)]";
    }
}
